go/chat/utils/utils.go

```diff
@@ -486,6 +488,21 @@ func FilterExploded(expunge *chat1.Expunge, msgs []chat1.MessageUnboxed, now tim
 	return res
 }
 
+func GetReaction(msg chat1.MessageUnboxed) (string, error) {
```

Comment:
not needed?

Reply:
Kept it for the future when the reaction is included in the toast.